plugin/federation/fedruntime/runtime.go

```diff
@@ -11,3 +11,6 @@ type Service struct {
 type Entity interface {
 	IsEntity()
 }
+
+// Used for the Link directive
+type Link interface{}
```

Comment:
I'm idly wondering if the Link interface should have functions to introspectively answer:
1. What version was imported (currently only "2.0" possible)
2. What imports out of the list: `["@key","@shareable", "@provides", "@external", "@tag", "@extends", "@override", "@inaccessible"]`